tests/specs/Code/Code_Inline_TextWrap_Always.txt

```diff
@@ -18,3 +18,24 @@ Testing
 
 [expect]
 Testing `this` out.
+
+!! should wrap inline code across lines !!
+Testing `this
+out`.
+
+[expect]
+Testing `this out`.
+
+!! should rewrap inline code across lines in blockquote !!
+> Testing `this
+> out`.
+
+[expect]
+> Testing `this out`.
+
+!! should rewrap inline code across lines which are too long in blockquote !!
+> This is some long text, which will have a bit of inline code across the `80 character limit` and then more after that.
+
+[expect]
+> This is some long text, which will have a bit of inline code across the `80
+> character limit` and then more after that.
```

Comment:
Discovered this failure mode while testing this PR against the Rust book, where there are a few of these. Should hopefully have a fix in for this tomorrow morning!

Reply:
Leaving some breadcrumbs here for myself, given I do not expect to get back to this till January—I spent some time chasing this (after beating my head against the wall that is getting useful debugging set up for Rust 😑) and concluded *so far* that it is down to something amiss in how `dprint-plugin-markdown` is using the result of parsing from `pulldown-cmark`, but haven’t gotten further than that.

Specifically, it ends up seeing an inline code snippet that wraps across a line in a blockquote as *including* the `>` character. It will correctly *introduce* the wrapping for a too-long line, but is not stable because once it has done so, on the next formatting pass it will then interpret it as part of the inline code instead of as a blockquote marker.

A side observation: this is just a challenging issue. Here is a simple sample that I used for testing `pulldown-cmark` itself locally:

```markdown
> This is some `inline
> code`. And it gets
> *rendered* just fine.
> The problem is thus
> not `pulldown-cmark`
> but something in how
> `dprint` uses it.
```

Notice how GitHub’s (line-wrapping-aware) renderer trips in a different way over the exact same thing, *not* wrapping when it *should*:

> This is some `inline
> code`. And it gets
> *rendered* just fine.
> The problem is thus
> not `pulldown-cmark`
> but something in how
> `dprint` uses it.